### DISCUSSION TITLE
以前版本的多选支持ctrl作为组合键应该怎么配置

### DISCUSSION THREAD

**Discussion by paiap** [didi/LogicFlow#1264] [Q&A] [answered]
![image](https://github.com/didi/LogicFlow/assets/128579430/06e491b2-35f5-439a-818e-e3bbe2c39d5a)

**Comment by wumail** [comment, marked as answer]
见https://docs.logic-flow.cn/docs/#/zh/api/logicFlowApi 中的multipleSelectKey配置项

**Reply by paiap** [reply]
感谢感谢，找着了！！！
